### PR TITLE
Update contribution guide

### LINKED ISSUE #494: Fix contribution guide.
### 🚀 Feature Request

We should write new contribution guide.

### Motivation

Our contribution guide is outdated.

### Proposal
Add section "Setting up your development environment".
Look at the section "Before the PR" and fix the links.
And make other small improvements to make the contribution guide simple, short and clear.

### Test cases

_No response_

### Alternatives

_No response_

### Additional context

_No response_

### Checklist

- [x] I discussed this issue with ETNA Team

### PR DESCRIPTION
**Pull request by Mr-Geekman**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Type of Change

- [x] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Proposed Changes
Look #494.

## Related Issue
#494.

## Closing issues
Closes #494.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,42 +9,71 @@ Every good PR usually consists of:
...
+Before writing any code it is useful to set up a development environment.
+1. Clone etna library to some folder and go inside:
+```bash
+git clone https://github.com/tinkoff-ai/etna.git etna/
+cd etna
+```
+2. Run installation with `poetry` ([poetry installation guide](https://python-poetry.org/docs/#installation)):
+```bash
+poetry install -E all-dev
+```
```
Comment (iKintosh) [inline CONTRIBUTING.md:32]: We should probably add section about creating new virtual environment.

**Review by iKintosh — COMMENTED** [review]
Looks a lot better than the current version!

Probably we should add that contributor could ask us for help if something is not clear.

File: CONTRIBUTING.md (hunk 1 above)
Comment (iKintosh) [reply CONTRIBUTING.md:32]: Something like: we recommend creating new virtual environment. At etna team we use pyenv, but you can use whatever tools you like.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,42 +9,71 @@ Every good PR usually consists of:
...
+### Tests
+
+Do not forget to check that your code passes the unit tests:
+```bash
+poetry install -E tests
+
+pytest tests -v
+pytest etna -v --doctest-modules
+```
+And code style checks:
```
Comment (iKintosh) [inline CONTRIBUTING.md:62]: Maybe something like this:
```suggestion
ETNA uses the [black](https://github.com/psf/black) and [flake8](https://github.com/pycqa/flake8) with several plugins for coding style checks as well as [mypy](https://github.com/python/mypy) for type checks and you must ensure that your code follows it. If not, the CI will fail and your Pull Request will not be merged.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -9,42 +9,71 @@ Every good PR usually consists of:
...
 
 If you are not familiar with creating a Pull Request, here are some guides:
-- http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request
-- https://help.github.com/articles/creating-a-pull-request/
+- [Creating a pull request](https://help.github.com/articles/creating-a-pull-request/);
+- [Creating a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+
+### Tests
+
+Do not forget to check that your code passes the unit tests:
```
Comment (iKintosh) [inline CONTRIBUTING.md:55]:
```suggestion
Do not forget to check that your code passes the unit tests. 
If not, the CI will fail and your Pull Request will not be merged.
```

**Review by iKintosh — COMMENTED** [review]
Added some details

File: CONTRIBUTING.md (hunk 1 above)
Comment (Mr-Geekman) [reply CONTRIBUTING.md:32]: As I remember, poetry creates environment by itself by default. May be we should add that to activate this environment contributor should run `poetry shell`?

Comment (iKintosh) [reply CONTRIBUTING.md:32]: Right! 
We use pyenv to set proper python version. 
So no section about creating new virtual environment is needed.

Comment (iKintosh) [reply CONTRIBUTING.md:32]: This conversation may be resolved.

**Review by iKintosh — DISMISSED** [review]

**Review by iKintosh — APPROVED** [review]

**Pull request merged**